src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,86 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException; // Правильний імпорт винятку
+import core.basesyntax.model.User;
+import org.junit.Before;
+import org.junit.Test;
```

Comment:
Replace `@Test` with `@org.junit.jupiter.api.Test` to be consistent with JUnit 5 conventions used in this test class.